node/pkg/watchers/evm/watcher.go

```diff
@@ -241,6 +259,58 @@ func (w *Watcher) Run(parentCtx context.Context) error {
 		} else {
 			w.logger.Info("assuming instant finality")
 		}
+
+		// Initialize a Transfer Verifier
+		if w.txVerifierEnabled {
+
+			// This shouldn't happen as Transfer Verification can
+			// only be enabled by passing at least one chainID as a
+			// CLI flag to guardiand, but this prevents the code
+			// from erroneously setting up a Transfer Verifier or
+			// else continuing in state where txVerifierEnabled is
+			// true but the actual Transfer Verifier is nil.
+			if !slices.Contains(txverifier.SupportedChains(), w.chainID) {
+				return errors.New("watcher attempted to create Transfer Verifier but this chainId is not supported")
+			}
+
+			// TODO: It would be better to avoid hard-coding the Wrapped Native Addresses.
+			var tbridge eth_common.Address
+			var weth eth_common.Address
+			switch w.env {
+			case common.UnsafeDevNet:
+				tbridge = eth_common.BytesToAddress(sdk.KnownDevnetTokenbridgeEmitters[w.chainID])
+				weth = eth_common.HexToAddress("0xDDb64fE46a91D46ee29420539FC25FD07c5FEa3E")
```

Comment:
Re the TODO, how about creating a set of `Known[X]NetWETHAddress` constants in the SDK alongside the existing constants?